spec/features/shift_patterns_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature 'shift patterns' do
```

Comment:
Now these feature specs are becoming more meaningful we should adopt the same style as the planner project's features. I.e expressing the steps in the given-when-then style.